src/test/java/org/javamrt/mrt/RipeRISDumpTest.java

```diff
@@ -0,0 +1,31 @@
+package org.javamrt.mrt;
+
+import org.testng.annotations.Test;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.assertNull;
+
+import java.io.*;
+
+public class RipeRISDumpTest {
+    @Test
+    public void testParseDump()
+    {
+        Exception exception = null;
+        try (InputStream input =
+                getClass().getResourceAsStream("/updates.20170103.0630")) {
+            BGPFileReader bgpFileReader =
+                new BGPFileReader(new BufferedInputStream(input));
+            MRTRecord mrtRecord = null;
+            while (true) {
+                mrtRecord = bgpFileReader.readNext();
```

Comment:
(Suggestion) Maybe check for expected content as well rather than just see if it parses?